pages/common/keep-header.md

```diff
@@ -0,0 +1,20 @@
+# keep-header
+
+> Convenience utility that runs Unix commands in a header-aware fashion.
```

Comment:
What does this do? After reading all of the command examples, I still have no idea.

From what I gather, it doesn't pipe the first line of a file/text to a command

Reply:
It pipes line 1 directly to stdout

Line 2 to EOF are passed to command

Basically, line 1 is short-circuited to stdout

Does this make sense now?

Reply:
I think I now understand, it just allows me to make adequate suggestions, thanks for the help.